README.md

```diff
@@ -357,6 +357,25 @@ grafana_dashboard { 'example_dashboard':
 `grafana_user` and `grafana_password` are optional, and required when
 authentication is enabled in Grafana.
 
+Example:
+Make sure the `grafana-server` service is up and running before creating the `grafana_dashboard` definition. One option is to use the `http_conn_validator` from the [healthcheck](https://forge.puppet.com/puppet/healthcheck) module
+
+```puppet
+http_conn_validator { 'grafana-conn-validator' :
+  host     => 'localhost',
+  port     => '3000',
+  use_ssl  => false,
+  test_url => '/public/img/grafana_icon.svg',
+  require  => Class['grafana'],
+} ->
```

Comment:
puppet-lint will complain about this, can you put '->' in front of 'grafana_dashboard{..' ?

Reply:
done 👍